frontend/src/components/Modals/VerifyModal.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+const VerifyModal = () => {
+  return (
+    <>
+      <div className="darkBG"/>
+      <div className="centered">
+        <div className="theModal">
```

Comment:
Perhaps its these two that can be one?

Reply:
I actually renamed this Modal to be ResponseModal.js. So I used this Modal like a successful confirmation for forgotPassword and resetPassword. Look at the RepsonseModal.js file, i replaced the context with a variable so that we can custom what ever on the Modal.